cli/Valet/PhpFpm.php

```diff
@@ -167,15 +186,57 @@ public function stopRunning()
         );
     }
 
+    /**
+     * Stop PHP, if a specific version isn't being used globally or by any sites.
+     *
+     * @param  string|null  $phpVersion
+     * @return void
+     */
+    public function maybeStop($phpVersion)
+    {
+        if (! $phpVersion) {
+            return;
+        }
+
+        if (strpos($phpVersion, 'php') === false) {
+            $phpVersion = 'php'.$phpVersion;
+        }
+
+        $phpVersion = $this->normalizePhpVersion($phpVersion);
+
+        if (! in_array($phpVersion, $this->utilizedPhpVersions())) {
+            $this->brew->stopService($phpVersion);
+        }
+    }
+
     /**
      * Use a specific version of php.
      *
      * @param $version
-     * @param $force
-     * @return string
+     * @param  bool  $force
+     * @param  string|null  $directory
+     * @return string|void
      */
-    public function useVersion($version, $force = false)
+    public function useVersion($version, $force = false, $directory = null)
     {
+        if ($directory) {
+            $site = $this->site->getSiteUrl($directory);
+
+            if (! $site) {
+                warning(sprintf('The [%s] site could not be found in valet site list.', $directory));
+                exit();
+            }
+
+            if ($version == 'default') { // Remove isolation for this site
+                $customPhpVersion = $this->site->customPhpVersion($site); // Example output: "74"
+                $this->site->removeIsolation($site);
+                $this->maybeStop($customPhpVersion);
+                $this->nginx->restart();
+                info(sprintf('The [%s] site is now using default php version.', $site));
+                exit();
```

Comment:
I think we'd want to `return` here instead of exiting, right?

```suggestion
                return;
```